tests/components/application-connector/test/compass-runtime-agent/testkit/oauth/types.go

```diff
@@ -0,0 +1,38 @@
+package oauth
+
+import "time"
+
+const (
+	contentTypeHeader                = "Content-Type"
+	contentTypeApplicationURLEncoded = "application/x-www-form-urlencoded"
+
+	grantTypeFieldName   = "grant_type"
+	credentialsGrantType = "client_credentials"
+
+	scopeFieldName = "scope"
+	scopes         = "application:read application:write formation:write runtime:read runtime:write"
```

Comment:
Should we have some `formation` scope?

Reply:
We have `formation:write` and it seems to work fine.